app/assets/javascripts/lib/vcalendar.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Create an vCalendar file with all the current courses on it,
+ * set to weekly, starting with the current week.
+ */
+var createVCalendar = function(periods) {
```

Comment:
Maybe it should be in the schedule folder but in a new file, not controller.js? Seems a little weird to throw it in with (presumably) global utilities.